Explorer/Assets/DCL/Infrastructure/Utility/UnityObjects/UnityObjectUtils.cs

```diff
@@ -19,6 +20,8 @@ void SetQuitting()
             {
                 isQuitting = true;
                 Application.quitting -= SetQuitting;
+                Debug.Log("Application is quitting");
```

Comment:
why is not our ReportHub?

Reply:
I think validation test won't pass because of this (but we can exclude this file from the test, if there is no way to use ReportHub)

Reply:
ReportHub LOG ALWAYS?

Reply:
Can't reference DCL.Optimization in Utility because cyclic dependency.

Reply:
But also, what does ReportHub have to do with optimization. But double also, what kind of assembly name is "optimization". Optimization is not a software component, it's an activity a human programmer does.

Reply:
Yep I've raised this before

I am ok with this workaround for now.... write a comment